docs/standard/linq/toc.yml

```diff
@@ -302,21 +302,21 @@
           href: program-nodes.md
         - name: Mixed declarative code-imperative code bugs
           href: mixed-declarative-code-imperative-code-bugs.md
-        - name: "How to: Stream XML fragments with access to header information"
+        - name: "How to stream XML fragments with access to header information"
           href: stream-xml-fragments-access-header-information.md
-        - name: "How to: Perform streaming transform of large XML documents"
-          href: perform-streaming-transform-large-xml-documents.md
-        - name: "How to: Read and write an encoded document"
+        - name: "How to do streaming transform of large XML documents"
```

Comment:
I know that Acrolinx complains about perform but I'd keep it here

Reply:
That's fine. I only made the change because you had made a perform -> do change in a previous PR.